graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -99,6 +100,8 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(Indices.class);
 
+    private static final String GRAYLOG_INTERNAL_TEMPLATE_NAME = "graylog-internal";
```

Comment:
I used that name to make it clear that this is our Graylog internal mapping and because users might have a template named "graylog" already.